nupic/research/connections.py

```diff
@@ -23,14 +23,28 @@
 
 
 
+class SynapseData(object):
+
+
+  __slots__ = ("segment", "presynapticCell", "permanence")
+
+
+  def __init__(self, segment, presynapticCell, permanence):
+    self.segment = segment
+    self.presynapticCell = presynapticCell
+    self.permanence = permanence
+
+
+  def __eq__(self, other):
+    return (self.segment, self.presynapticCell, self.permanence) == other
+
+
+
 class Connections(object):
   """
   Class to hold data representing the connectivity of a collection of cells.
   """
 
-  SynapseData = namedtuple("SynapseData", ["segment",
-                                           "presynapticCell",
-                                           "permanence"])
 
```

Comment:
This was raised in https://github.com/nupic-community/nupic.studio/issues/29#issuecomment-142504682 and recently updated with a new user facing this problem. I recreated this locally. Let me know if there is a simple Py change for this (potentially an isolated change just within nupic.studio to cope)?